Tiltfile

```diff
@@ -452,8 +457,9 @@ docker_build(
     dockerfile = "./ethereum/Dockerfile",
 
     # ignore local node_modules (in case they're present)
-    ignore = ["./ethereum/node_modules"],
+    ignore = ["./node_modules"],
```

Comment:
should this also ignore ./ethereum/node_modules ?

Reply:
I believe this is a bug that existed earlier. the context is './ethereum' so here it is ignoring './ethereum/node_modules'.

Earlier it was ignoring './ethereum/ethereum/node_modules'